### PR TITLE
Feature/86epu3gf5/logout button

### PR DESCRIPTION
**Pull request by JezChristian**
Reusable Dropdown
Logout Dropdown Relocation

### CODE REVIEW

**Review by LizterBabael — APPROVED** [review]

File: packages/core-library/components/GenericDropDown/DropDown.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    const open = Boolean(anchorEl);
+    const id = open ? 'simple-popper' : undefined;
+
+    return (
+        <Box >
+            <Button sx={{ gap: 2 }} aria-describedby={id} type="button" onClick={handleClick} variant="outlined"><AccountCircleIcon color="primary" fontSize="small" /> User </Button>
+            <Popper sx={{ zIndex: 1500, width: '150px', textAlign: 'center' }} id={id} open={open} anchorEl={anchorEl}>
+                <Box sx={{ boxShadow: 1, bgcolor: 'background.paper', zIndex: '1500', borderRadius: '5px', marginTop: 1 }}>
+                    {list.length > 0 && list.map((item, index) => (
```
Comment (JMSevilla) [inline packages/core-library/components/GenericDropDown/DropDown.tsx:28]: I see that you use mapping to make the list dynamic. but did you consider when we have its functionality ? it should be dynamic too right.

File: packages/core-library/components/GenericDropDown/MockDropDownMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {
+    AccountCircle as AccountCircleIcon,
+    Logout as LogoutIcon
+} from "@mui/icons-material";
+import { NavigationType } from '../../types/navigation';
+
+export const DropDownMenu: NavigationType[] = [
```
Comment (JMSevilla) [inline packages/core-library/components/GenericDropDown/MockDropDownMenu.tsx:7]: All utilization must be placed within the project core folder.

**Review by JMSevilla — CHANGES_REQUESTED** [review]

**Review by acostales08 — APPROVED** [review]

**Review by DropitupQwerty — APPROVED** [review]

File: apps/web-backoffice-generic/src/core/constant/UserDropDown.tsx [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+const handleLogout = () => { }
+
 export const DropDownMenu: NavigationType[] = [
     {
         id: 1,
         label: "Logout",
         icon: <LogoutIcon fontSize="small" />,
-        onClick: () => { console.log("Logout Clicked") }
-
+        onClick: handleLogout
```
Comment (JMSevilla) [inline apps/web-backoffice-generic/src/core/constant/UserDropDown.tsx:11]: @JezChristian we can't also accept this. imagine if we store this on our DB.

**Review by JMSevilla — CHANGES_REQUESTED** [review]

File: apps/web-backoffice-generic/src/pages/shared/Layout.tsx [hunk 4]
```diff
@@ -24,13 +24,15 @@ const Layout: React.FC<React.PropsWithChildren<Props>> = ({
   const theme = useTheme();
   const mockMenu = mockMenus(isAuthenticated);
 
+  const onLogout = () => { console.log("LOGOUT") }
```
Comment (JMSevilla) [inline apps/web-backoffice-generic/src/pages/shared/Layout.tsx:27]: What i mean by this is that you should use our existing logout from useAuthContext.

**Review by JMSevilla — CHANGES_REQUESTED** [review]

**Review by JMSevilla — APPROVED** [review]

**Pull request merged**